super-agent/super-agent.js

```diff
@@ -0,0 +1,24 @@
+// https://codeforces.com/problemset/problem/12/A
+
+let input = ".XX...XX."
+
+function isThePasswordSymmetric (input) {
+    let n = 0
+    let centralButton = Math.floor(input.length/2)
```

Comment:
input.length powinno byc wyciagniete do stalej. Bo wyciagasz to dwa razy niepotrzebnie